timingApp/Db/event_log.db

```diff
@@ -223,6 +241,11 @@ record(compress, "$(P)$(R)EVENTbuffer"){
   field(NSAM, "1000")
   field(INP, "$(P)$(R)LOGEVENT")
   field(FLNK, "$(P)$(R)LOGSOFTCNT")
+  info(Q:group, {
+    "$(P)$(R)EventTable-Mon": {
+      "value.B": {+type: "plain", +channel: "VAL", +trigger: "*"}
```

Comment:
I'm not sure if that definition of trigger should be mentioned in the commit message. By following the PVXS, I could understand why it is defined here. Maybe that's still something useful for git-blaming in the future?

Reply:
I considered it self explanatory if the docs are read. We are using records from a single processing chain, so the last record in the chain is the one with `+trigger: "*"`.

Reply:
Makes sense.